merlin/main.py

```diff
@@ -312,8 +319,67 @@ def process_example(args):
         setup_example(args.workflow, args.path)
 
 
+def check_merlin_status(args, spec):
```

Comment:
What do you think of moving status/monitor logic to `router.py` or elsewhere? Would that make more sense than `main.py`?

Reply:
The reason it is in main is because it doesn't require the router disconnect like the other router functions. If this is an aesthetic response, I don't have an issue with moving the function.

Reply:
Yes it's aesthetic, this just seems a little different from the other logic in main.